storyblok/storyblok-design-system
# bugfix(int-377): Removing border + Removing automatic aperture from slide over

## Pull request type

Jira Link: [INT-622](https://storyblok.atlassian.net/browse/INT-622) + [INT-377](https://storyblok.atlassian.net/browse/INT-377)




## How to test this PR

In the video below you can see that I show the two problems that were solved in this PR, one is the double border in the date piker time selection and the other is that the lside over opened automatically, and that was removed.

For more information please check the task descriptions.

https://user-images.githubusercontent.com/40925579/198716362-6a2e6180-7719-4161-9b63-56f584725282.mov


## What is the new behavior?

- Slideover does not automatically open in documentation
- Removed the double border appearance of the time picker in datepicker

## Other information
